examples-heron/src/main/java/org/openconnectors/pulsar/PulsarSinkTopology.java

```diff
@@ -0,0 +1,29 @@
+package org.openconnectors.pulsar;
+
+import com.twitter.heron.streamlet.*;
+
+import static org.openconnectors.util.DataUtils.randomFromList;
+import static org.openconnectors.util.DataUtils.sentences;
+
+public class PulsarSinkTopology {
+    private PulsarSinkTopology() { }
+
+    public static void main(String[] args) throws Exception {
+
+        Builder processingGraphBuilder = Builder.createBuilder();
+        processingGraphBuilder
+                .newSource((SerializableSupplier<byte[]>) () -> {
```

Comment:
don't need a thread for this

Reply:
take a look at KafkaSinkTopology

Reply:
its not a thread just implementing the function